tasks/src/sentence-similarity/data.ts

```diff
@@ -66,25 +66,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["sentence-similarity"],
 	metrics:   [
 		{
-			description: "The reciprocal rank is a measure used to rank the relevancy of documents given a set of documents. Reciprocal Rank is the reciprocal of the rank of the document retrieved, meaning, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
+			description: "Reciprocal ranking is a measure used to rank the relevance of documents given a set of documents. The Reciprocal Rank is the reciprocal of the retrieved document rank, that is, if the rank is 3, the Reciprocal Rank is 0.33. If the rank is 1, the Reciprocal Rank is 1.",
 			id:          "Mean Reciprocal Rank",
 		},
 		{
-			description: "The similarity of embeddings is evaluated mainly on cosine similarity. It’s calculated as the cosine of the angle between two vectors. It is particularly useful when your documents do not have the same length.",
+			description: "The similarity of the embeddings is evaluated mainly based on the similarity of the cosine. It is calculated as the cosine of the angle between two vectors. It is particularly useful when your texts are not the same length.",
 			id:          "Cosine Similarity",
 		},
 	],
 	models: [
 		{
-			description: "This is a good model that works for sentences and paragraphs and can be used for clustering and semantic search.",
+			description: "This is a model that works well for sentences and paragraphs and can be used for grouping and semantic searches.",
 			id:          "sentence-transformers/all-mpnet-base-v2",
 		},
 		{
 			description: "A multilingual model trained for FAQ retrieval.",
 			id:          "clips/mfaq",
 		},
 	],
-	summary:      "Sentence similarity is the task of determining how similar two texts are. Sentence similarity models conver texts into embeddings and calculate the similarity between these embeddings. This task is particularly useful for semantic search and clustering. Sentence similarity has various subtasks, such as passage ranking or semantic textual similarity.",
```

Comment:
We're keeping the model and dataset descriptions very short, please keep it in mind :)

Reply:
Agree, will make it shorter. Thanks!